scripts/verify-terraform-version.sh

```diff
@@ -2,7 +2,7 @@
 
 tf_version=$(terraform --version)
 
-if [[ ${tf_version} != *"1.4.5"* ]]; then
+if [[ ${tf_version} != *"1.4.6"* ]]; then
   echo "Please set your terraform version to 1.4.5 before deploying."
```

Comment:
Nit: I think this should say 1.4.6 now 😄

Reply:
Apologies, that wasn't a super clear comment. I meant the echo statement says to set your version to `1.4.5`, I think it should say `1.4.6` now!

Reply:
Geez.. you're right.. smh